content/hi/docs/concepts/overview/components.md

```diff
@@ -0,0 +1,130 @@
+स्वतंत्र नियंत्रण एक एकल बाइनरी में लूप करता है जिसे आप एकल प्रक्रिया के रूप में चलाते हैं। तुम कर सकते हो
+प्रदर्शन में सुधार करने या विफलताओं को सहन करने में सहायता के लिए क्षैतिज रूप से स्केल करें (एक से अधिक प्रतिलिपि चलाएं)।
+
+निम्नलिखित नियंत्रकों में क्लाउड प्रदाता निर्भरताएँ हो सकती हैं:
+
+  * नोड नियंत्रक: क्लाउड प्रदाता की जांच के लिए यह निर्धारित करने के लिए कि क्लाउड में नोड को प्रतिक्रिया देना बंद करने के बाद हटा दिया गया है या नहीं
+  * मार्ग नियंत्रक: अंतर्निहित क्लाउड अवसंरचना में मार्ग स्थापित करने के लिए
+  * सर्विस कंट्रोलर: क्लाउड प्रोवाइडर लोड बैलेंसर्स को बनाने, अपडेट करने और हटाने के लिए
+
+## Node Components
+
+नोड घटक प्रत्येक नोड पर चलते हैं, रनिंग पॉड्स को बनाए रखते हैं और कुबेरनेट्स रनटाइम वातावरण प्रदान करते हैं।
+
+## kubelet
+
+{{< glossary_definition term_id="kubelet" length="all" >}}
+
+### kube-proxy
+
+{{< glossary_definition term_id="kube-proxy" length="all" >}}
+
+### Container runtime
+
+{{< glossary_definition term_id="container-runtime" length="all" >}}
+
+## Addons
+
+एडॉन्स क्लस्टर सुविधाओं को लागू करने के लिए कुबेरनेट्स संसाधनों ({{< glossary_tooltip term_id="daemonset" >}},
+{{< glossary_tooltip term_id="deployment" >}}, आदि)
+ का उपयोग करते हैं। क्योंकि ये क्लस्टर-स्तर की सुविधाएँ प्रदान 
+ कर रहे हैं, ऐड-ऑन के लिए नामित संसाधन `क्यूब-सिस्टम` नेमस्पेस के अंतर्गत आते हैं।
+
+चयनित योजकों का वर्णन नीचे किया गया है; उपलब्ध एडॉन्स की विस्तृत सूची के लिए, 
+कृपया [एडॉन्स](/docs/concepts/cluster-administration/addons/) देखें।
+
+### DNS
+
+जबकि अन्य एडोनों की सख्ती से आवश्यकता नहीं है, सभी कुबेरनेट समूहों में [क्लस्टर डीएनएस](/docs/concepts/services-networking/dns-pod-service/) होना चाहिए, क्योंकि कई उदाहरण इस पर भरोसा करते हैं।
+
+क्लस्टर डीएनएस एक डीएनएस सर्वर है, जो आपके वातावरण में अन्य डीएनएस सर्वर के अलावा कुबेरनेट्स सेवाओं के लिए डीएनएस रिकॉर्ड प्रदान करता है।
+
+कुबेरनेट्स द्वारा शुरू किए गए कंटेनर स्वचालित रूप से इस डीएनएस सर्वर को अपनी डीएनएस खोजों में शामिल करते हैं।
+
+### Web UI (Dashboard)
+
+[डैशबोर्ड](docs/tasks/access-application-cluster/web-ui-dashboard/) कुबेरनेट्स समूहों के लिए एक सामान्य उद्देश्य, वेब-आधारित यूआई है। यह उपयोगकर्ताओं को क्लस्टर में चल रहे एप्लिकेशन के साथ-साथ स्वयं क्लस्टर का प्रबंधन और समस्या निवारण करने की अनुमति देता है।
+
+### Container Resource Monitoring
+
+[कंटेनर रिसोर्स मॉनिटरिंग](/docs/tasks/debug/debug-cluster/resource-usage-monitoring/) एक केंद्रीय डेटाबेस में कंटेनरों 
+के बारे में सामान्य समय-श्रृंखला मेट्रिक्स रिकॉर्ड करता है, और उस डेटा को ब्राउज़ करने के लिए यूआई प्रदान करता है।
+
+### Cluster-level Logging
+
+खोज/ब्राउज़िंग इंटरफ़ेस के साथ एक केंद्रीय लॉग स्टोर में 
+कंटेनर लॉग को सहेजने के लिए एक [क्लस्टर-स्तरीय लॉगिंग](/docs/concepts/cluster-administration/logging/) तंत्र जिम्मेदार है।
+
+## {{% heading "whatsnext" %}}
+
+[नोड्स](docs/concepts/architecture/nodes/) के बारे में जानें
+[नियंत्रकों](docs/concepts/architecture/controller/) के बारे में जानें
+[क्यूब-शेड्यूलर](docs/concepts/scheduling-eviction/kube-scheduler/) के बारे में जानें
```

Comment:
```suggestion
[kube-scheduler](docs/concepts/scheduling-eviction/kube-scheduler/) के बारे में जानें
```

Reply:
If Pods are translated into Hindi, this should be too. Shouldn't it be?

Reply:
AFAIT since it a part of control plane components and people can more easily associate when it (kube-scheduler) is used as it in the documentation, I feel it better to keep `kube-scheduler` as it is. Plus other localization like `zh-cn`, `ko`, `ja` and `pt-br` have also kept it as it is as well. (I think they have kept `etcd` as it in the documentation as well [though I might be wrong on this case])